src/applications/personalization/profile/components/notification-settings/NotificationChannel.jsx

```diff
@@ -240,6 +238,7 @@ const mapStateToProps = (state, ownProps) => {
     isOptedIn: channel.isAllowed,
     isMissingContactInfo,
     permissionId: channel.permissionId,
+    defaultSendIndicator: channel?.defaultSendIndicator,
```

Comment:
not in this case since `defaultSendIndicator` may or may not be present at all depending on the notification, but the rest will definitely be present.